src/featureFlags/dynamicConfig/quickRouteChains.ts

```diff
@@ -1,13 +1,20 @@
 import { ChainId } from '@uniswap/sdk-core'
+import { useFeatureFlagsContext } from 'featureFlags'
 
 import { DynamicConfigName, useDynamicConfig } from '.'
 
-// eslint-disable-next-line import/no-unused-modules
 export function useQuickRouteChains(): ChainId[] {
-  const config = useDynamicConfig(DynamicConfigName.quickRouteChains)
-  const chains = config.get('chains', [])
+  const statsigConfig = useDynamicConfig(DynamicConfigName.quickRouteChains)
+  const featureFlagsContext = useFeatureFlagsContext()
+  let chains = statsigConfig.get(DynamicConfigName.quickRouteChains, []) as ChainId[]
+
+  const modalSettings = featureFlagsContext.configs[DynamicConfigName.quickRouteChains]
+  if (modalSettings) {
+    const modalSetChains = modalSettings[DynamicConfigName.quickRouteChains]
+    if (Array.isArray(modalSetChains) && modalSetChains !== chains) chains = modalSetChains
+  }
```

Comment:
i think this block can be clearer without a `let` statement and to be more descriptive of whats going on 🤔 

```
const remoteConfigChains = statsigConfig.get(DynamicConfigName.quickRouteChains, []) as ChainId[]
const localConfigChains = featureFlagsContext.configs[DynamicConfigName.quickRouteChains]?.[DynamicConfigName.quickRouteChains] 

const chains = Array.isArray(localConfigChains) ? localConfigChains : remoteConfigChains
```

Reply:
ah this is much clearer thank you